lib/drivers/medtronicDriver.js

```diff
@@ -428,33 +451,32 @@ module.exports = function (config) {
                     return callback(err, null);
                   }
                   if (result) {
-                    if(result.resultsInReply) {
 
-                      var header = struct.extractBytes(result.bytes, 0, 2);
-                      console.log('Last header: ', common.bytes2hex(header));
-                      if(header[0] !== 0x51 && header[1] !== 0x01) {
+                    var header = struct.extractBytes(result.bytes, 0, 2);
+                    console.log('Last header: ', common.bytes2hex(header));
```

Comment:
I was thinking of leaving it in until after alpha testing, to make it easier to debug any issues. That reminds me that I should tell Brandon so that he doesn't start measuring upload times as part of alpha testing, as the logging as a big effect on time to upload.